cmake/CPackConfig.cmake

```
@@ -0,0 +1,11 @@
+set(CPACK_PACKAGE_NAME ${PROJECT_NAME})
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "SSE cpack")
+set(CPACK_PACKAGE_DESCRIPTION "This is a SSE cpack exercise submission")
+set(CPACK_PACKAGE_VENDOR "Kujoen")
+set(CPACK_PACKAGE_CONTACT "st141408@stud.uni-stuttgart.de")
+set(CPACK_PACKAGE_MAINTAINERS "Andreas Farley ${CPACK_PACKAGE_CONTACT}")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://github.com/Kujoen/cpack-exercise-wt2223")
```

```suggestion
set(CPACK_PACKAGE_HOMEPAGE_URL "https://github.com/Kujoen/cpack-exercise-wt2223")
set(CPACK_STRIP_FILES TRUE)
```
to strip more (all?) symbols